chakraTheme/components/Input.ts

```diff
@@ -0,0 +1,14 @@
+import cp from '@core-ds/primitives';
+
+const Input = {
```

Comment:
Is there an existing type you could use here? Typescript will infer the type when `Input` is used somewhere else. But within this file it's basically `any`.

Reply:
I don't think so,  chakra doesn't type these https://github.com/chakra-ui/chakra-ui/blob/main/packages/theme/src/components/input.ts

Reply:
I spoke with Rich about today and he had some good insight on ways to leverage types. I'd like to explore this again later as we get our themes more established.